distribution/docker/build.gradle

```diff
@@ -173,12 +189,16 @@ void addBuildDockerImage(final boolean oss) {
 }
 
 for (final boolean oss : [false, true]) {
-  addCopyDockerContextTask(oss)
-  addBuildDockerImage(oss)
+  for (final boolean ubi : [false, true]) {
+    addCopyDockerContextTask(oss, ubi)
+    addBuildDockerImage(oss, ubi)
+  }
 }
 
 assemble.dependsOn "buildOssDockerImage"
 assemble.dependsOn "buildDockerImage"
+assemble.dependsOn "buildOssUbiDockerImage"
+assemble.dependsOn "buildUbiDockerImage"
```

Comment:
Could we move this stuff into `addBuildDockerImage()`?

Reply:
I pushed 5e4e8ecef63594c75193c9f3a5db82b72639fcd2.